src/components/navigation.component.js

```diff
@@ -93,15 +95,25 @@ class Navigation extends LitElement {
             <ul>
               ${!this.isToken
                 ? html`
-                    <li><a href="/">Home</a></li>
+                    <li class="active"><a href="/">Home</a></li>
                     <li><a href="/sign-in">Sign in</a></li>

```

Comment:
For the routes, lets create a separate constants file(for e.g routeConstants) and import them here so that we can modify them from a single source if any changes are to made in the future